src/codecatalyst/vue/create/source.vue

```diff
@@ -48,7 +48,8 @@
                 <input
                     id="branch-input"
                     type="text"
-                    placeholder="branch-name"
+                    :placeholder="newBranchNamePlaceholder"
```

Comment:
If it's disabled can we show something like: 

    Cannot create new branch for a linked (third-party) repository

Reply:
This is too long, but I can merge this + what Jo decided: 'Not Applicable for Linked Repo'